### DISCUSSION TITLE
Can't export JupyterLite notebooks

### DISCUSSION THREAD

**Discussion by guyfriedman2001** [jupyterlite/jupyterlite#1421] [Q&A]
I use JupyterLite on my Ipad for studying 'on the fly', but when I finish a task or project, I cant seem to be able to export the notebook to be able to send to someone else/ continue on the computer/ save to my files etc... 
Anyone else experienced this problem ot knows how to solve it?

Photo description:
The dropdown menu - 
<img width="1180" alt="image" src="https://github.com/jupyterlite/jupyterlite/assets/130835393/afbe455c-5a23-477f-bcc9-2d5da64fefd3">

After I press "Export file as..." -
<img width="1180" alt="image" src="https://github.com/jupyterlite/jupyterlite/assets/130835393/171584ed-5f94-4639-b0f6-839f9ffa0634">

As you cann see, the option turns gray and I dont get any additional menu or options.

How to replicate:
Open the link - 
https://jupyter.org/try-jupyter/lab/index.html

Open a new notebook, write some code and try to export.

System: 
Ipad air 5 (2022)
Chrome for ipad os

**Comment by jtpio** [comment]
Thanks @guyfriedman2001.

It looks like there should be a "Download" entry in the File menu normally. Maybe we should look into this if it is a real issue.

Otherwise you should be able to download the file by right clicking on the file in the file browser:

![image](https://github.com/jupyterlite/jupyterlite/assets/591645/442f2fe3-443b-4a3a-82d2-6012e697df7b)

"Save and Export Notebook As..." is expected to not work for now. See https://github.com/jupyterlite/jupyterlite/issues/543 and related issues for more details.

**Reply by guyfriedman2001** [reply]
<img width="820" alt="image" src="https://github.com/jupyterlite/jupyterlite/assets/130835393/beafd39e-5ab0-42c3-acc9-0c82502f92c2">
I don't have the Download optin, and I can't right click because I dont have a mouse connected to my Ipad, is there a different way to do it?

**Reply by jtpio** [reply]
> I can't right click because I dont have a mouse connected to my Ipad, is there a different way to do it? 

Maybe with a long press?

Otherwise the missing "Download" menu entry should be fixed in JupyterLite.

**Reply by guyfriedman2001** [reply]
I tried long pressing and it didn't work, I'm currently trying to maybe solve the issue by installing 'nbconvert' using 'piplite'.
Although I'm still struggling a bit with 'piplite' (not sure about commands).

**Comment by Hias59** [comment]
In JupyterLite you need to click on 
![image](https://github.com/user-attachments/assets/0ec947c7-9984-4a21-9ae3-163f9d6f3c93) "Notebook with an arrow", then a new window of Jupyter opens, that allows to access this "download" menu!
